modules/oe_content_publication/oe_content_publication.post_update.php

```diff
@@ -29,3 +32,74 @@ function oe_content_publication_post_update_00001_update_field_labels(array &$sa
 function oe_content_publication_post_update_00002() {
   \Drupal::service('module_installer')->install(['oe_content_documents_field']);
 }
+
+/**
+ * Create the new fields in the Publication content type.
+ */
+function oe_content_publication_post_update_00003(): void {
+  $path = drupal_get_path('module', 'oe_content_publication');
+
+  $storage = new FileStorage($path . '/config/post_updates/00003_create_fields');
+  \Drupal::service('config.installer')->installOptionalConfig($storage);
+}
+
+/**
+ * Update field labels.
+ */
+function oe_content_publication_post_update_00004(array &$sandbox): void {
+  $new_field_labels = [
+    'node.oe_publication.oe_publication_type' => 'Resource type',
+    'node.oe_publication.oe_documents' => 'Files',
+  ];
+  foreach ($new_field_labels as $id => $new_label) {
+    $field_config = FieldConfig::load($id);
+    $field_config->setLabel($new_label);
+    $field_config->save();
+  }
+
+  $new_translatable_settings = [
+    'node.oe_publication.oe_summary' => FALSE,
+    'node.oe_publication.oe_teaser' => FALSE,
+  ];
+  foreach ($new_translatable_settings as $id => $value) {
+    $field_config = FieldConfig::load($id);
+    $field_config->setTranslatable($value);
+    $field_config->save();
+  }
+}
+
+/**
+ * Update Publication node form display.
+ */
+function oe_content_publication_post_update_00005(): void {
+  $path = drupal_get_path('module', 'oe_content_publication');
```

Comment:
Please use `new FileStorage(drupal_get_path('module', 'oe_content_publication') . '/config/post_updates/00005_update_form_display');` for consistency with another modules.

Reply:
This is fixed.